Docs: fix missed properties in document of `automation`, `network`, `fluidrelay server`, etc. (#19913)
hashicorp/terraform-provider-azurerm: website/docs/r/automation_module.html.markdown

* `automation_account_name` - (Required) The name of the automation account in which the Module is created. Changing this forces a new resource to be created.

> * `module_link` - (Required) The published Module link `module_link` block defined as blow.

Reviewer: ```suggestion
* `module_link` - (Required) A `module_link` block as defined below.
```

---

The `module_link` block supports the following:

* `uri` - (Required) The URI of the module content (zip or nupkg).

> * `hash` - (Optional) A `hash` block defined as below.

Reviewer: ```suggestion
* `hash` - (Optional) A `hash` block as defined below.
```
